src/Alpacka.Lib/Pack/Config/ModpackConfig.cs

```diff
@@ -42,5 +40,18 @@ public static ModpackConfig LoadYAML(string path)
             
             return config;
         }
+        
+        
+        public void SaveYAML(string path)
+        {
+            if (!Directory.Exists(path))
+                Directory.CreateDirectory(path);
```

Comment:
It's safe to call `CreateDirectory` if the directory already exists so you can remove the check.